test/blockchain.cpp

```diff
@@ -616,7 +614,28 @@ void updatePoW(BlockInfo& _bi)
 	_bi.noteDirty();
 }
 
-void writeBlockHeaderToJson(mObject& _o, BlockInfo const& _bi)
+mArray writeTransactionsToJson(Transactions const& txs)
+{
+	mArray txArray;
+	for (auto const& txi: txs)
+	{
+		mObject txObject;
+		txObject["nonce"] = toString(txi.nonce());
+		txObject["data"] = "0x" + toHex(txi.data());
+		txObject["gasLimit"] = toString(txi.gas());
+		txObject["gasPrice"] = toString(txi.gasPrice());
+		txObject["r"] = "0x" + toString(txi.signature().r);
+		txObject["s"] = "0x" + toString(txi.signature().s);
+		txObject["v"] = to_string(txi.signature().v + 27);
+		txObject["to"] = txi.isCreation() ? "" : toString(txi.receiveAddress());
+		txObject["value"] = toString(txi.value());
+		txObject = ImportTest::makeAllFieldsHex(txObject);
```

Comment:
here transaction object will be overwritten
need merge with https://github.com/ethereum/cpp-ethereum/pull/1609